.github/actions/deploy-action.yaml

```diff
@@ -0,0 +1,30 @@
+name: 'Deploy to Kalm'
+description: 'call deploy callbacks in Kalm'
+inputs:
+  KALM_API_ADDRESS:  # id of input
+    description: 'kalm api address'
+    required: true
+  KALM_DEPLOY_KEY:
+    description: ''
+    required: true
+  KALM_APP:
+    description: ''
+    required: true
+  KALM_COMPONENT:
+    description: ''
+    required: true
+  KALM_COMPONENT_IMG_TAG:
+    description: ''
+    required: true
+# outputs:
```

Comment:
Should we display some useful result that is returned from webhook api?

Reply:
add response as outputs